meson.build

```diff
@@ -1,33 +1,39 @@
-# Define project name
-project ('com.github.timecraft.js-test', 
-    'vala', 'c',
-    version: '0.1.0'
-)
-
+project ('com.github.timecraft.js-test', 'vala', 'c', version: '0.1.0')
```

Comment:
I personally find all of this on one line to be a bit harder to follow and not as nice

Reply:
Understand your point. I just always seen it like this in other elementary apps and it's formatted this way in the Getting Started guide.